src/_includes/layouts/form-vacature.liquid

```diff
@@ -49,11 +49,11 @@
                     </p>
                     <p class="form-element">
                     <label for="form-1-input-5" class="form-label">Factuuradres</label>
-                    <textarea name="Factuuradres" id="form-1-input-5" cols="50" rows="5" class="small" required></textarea>
+                    <textarea name="Factuuradres" id="form-1-input-5" cols="50" rows="5" class="small" autocomplete="street-address" required></textarea>
                     </p>
                     <p class="form-element">
                     <label for="form-1-input-6" class="form-label">E-mailadres voor de factuur</label>
-                    <input type="text" name="E-mailadres_voor_de_factuur" id="form-1-input-6" class="text" required>
+                    <input type="text" name="E-mailadres_voor_de_factuur" id="form-1-input-6" class="text" autocomplete="email work" required>
```

Comment:
Same as above: I think it's supposed to be order-dependent and thus `work email`.